signac/contrib/project.py

```diff
@@ -87,6 +108,23 @@ def _resolve_statepoint_filter(self, q):
                 yield 'statepoint.{}'.format(k), v
 
     def find_job_ids(self, filter=None, doc_filter=None):
+        """Find job ids from a state point or document filter.
+
+        Parameters
+        ----------
+        filter : dict
+            A mapping of key-value pairs that all indexed job statepoints are
```

Comment:
There's some inconsistency with statepoint and state point. Do they mean different things?

Reply:
line 1039 is "state point", but here is "statepoint"

Reply:
@jennyfothergill We generally try to use "state point" as two words, but this is not consistent across the docs. The property is named "statepoint", which is a source of confusion.

Reply:
```suggestion
            A mapping of key-value pairs that all indexed job state points are
```

Reply:
changed `"statepoint"/ "statepoints"` to `"state point"/ "state points"` at all the required places.